src/lib/common/macroSubstitute.cpp

```diff
@@ -145,7 +145,7 @@ bool macroSubstitute(std::string* to, const std::string& from, const Entity& en)
   //
   if (from.size() > outReqMsgMaxSize)
   {
-    LM_W(("Runtime Error (too large initial string, before substitution)"));
+    LM_E(("Runtime Error (too large initial string, before substitution)"));
```

Comment:
Increased to ERROR due to this is a problem that should be review from an operational point of view (probably enlarging the `-outReqMsgMaxSize` CLI)

NTC (informative)